### PR TITLE
fix(refs T36753): use full width to display the stn text

### PR DESCRIPTION
**Pull request by salisdemos**
**Ticket:** https://yaits.demos-deutschland.de/T36753

As not logged in user, write a statement a proceed to the "recheck"-view in the statement modal. There, the statement text should be displayed and readable.

### PR Checklist
- [x] Link all relevant tickets
- [x] Move the tickets on the board accordingly

### CODE REVIEW

**Review by spiess-demos — COMMENTED** [review]
doesn't the div take the full width by default if not told otherwise when being set to `display: block;`?

**Comment by salisdemos** [comment]
> doesn't the div take the full width by default if not told otherwise when being set to `display: block;`?

true

File: client/js/components/statement/publicStatementModal/StatementModalRecheck.vue [hunk 1]
```diff
@@ -218,7 +218,7 @@
       </span>
 
       <div
-        :class="prefixClass('sm:h-9 overflow-auto w-2')"
+        :class="prefixClass('sm:h-9 overflow-auto w-full')"
```
Comment (salisdemos) [inline client/js/components/statement/publicStatementModal/StatementModalRecheck.vue:221]:
```suggestion
        :class="prefixClass('sm:h-9 overflow-auto')"
```

**Review by spiess-demos — APPROVED** [review]
lgtm!

**Pull request merged**